source/develop/dev-process/working-with-gerrit.md

```diff
@@ -230,23 +234,15 @@ The patch life cycle process comprises of the following steps:
 
 ### Submitting a Topic Branch to Gerrit
 
-A [topic branch](http://progit.org/book/ch3-4.html) is a short-lived branch that you create and use for a single particular feature or related work.
+A [feature branch](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) is a short-lived branch that you create and use for a single particular feature or related work.
```

Comment:
you changed it because that link is broken right? in that case it can be replaced with https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows maybe